source/_docs/automation/templating.markdown

```
@@ -1,30 +1,36 @@
-title: "Automation trigger variables"
-description: "List all available variables made available by triggers."
+title: "Automation Templates"
+description: "List all trigger variables available to templates."
 ---
 
-Automations support [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger` and `this` template variables are available.
+Automations support the advanced features of [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger` and `this` template variables are available for automations.
 
-The template variable `this` is also available when evaluating any `trigger_variables` declared in the configuration.
+Example of variables used in templates:
+```jinja
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)